src/proposals/20251002/SparkEthereum_20251002.t.sol

```diff
@@ -0,0 +1,352 @@
+            vault_:     Ethereum.SPARK_VAULT_V2_SPUSDT,
+            minVsr:     1e27,
+            maxVsr:     TEN_PCT_APY,
+            depositCap: 50_000_000e6,
+            amount:     1_000_000e6
+        });
+    }
+
+    function test_ETHEREUM_sparkVaultsV2_configureSPETH() public onChain(ChainIdUtils.Ethereum()) {
+        _testVaultConfiguration({
+            vault_:     Ethereum.SPARK_VAULT_V2_SPETH,
+            minVsr:     1e27,
+            maxVsr:     FIVE_PCT_APY,
+            depositCap: 10_000e18,
+            amount:     1_000e18
+        });
+    }
+
+    function _testVaultConfiguration(
+        address vault_,
+        uint256 minVsr,
+        uint256 maxVsr,
+        uint256 depositCap,
+        uint256 amount
+    ) internal {
+        SparkLiquidityLayerContext memory ctx = _getSparkLiquidityLayerContext();
+
+        ISparkVaultV2 vault = ISparkVaultV2(vault_);
+
+        bytes32 takeKey = RateLimitHelpers.makeAssetKey(
+            MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_SPARK_VAULT_TAKE(),
+            vault_
+        );
+        bytes32 transferKey = RateLimitHelpers.makeAssetDestinationKey(
+            MainnetController(Ethereum.ALM_CONTROLLER).LIMIT_ASSET_TRANSFER(),
+            vault.asset(),
+            vault_
+        );
+
+        assertEq(vault.hasRole(vault.DEFAULT_ADMIN_ROLE(), Ethereum.SPARK_PROXY), true);
+        assertEq(vault.hasRole(vault.SETTER_ROLE(), Ethereum.ALM_OPS_MULTISIG),   false);
+        assertEq(vault.hasRole(vault.TAKER_ROLE(), Ethereum.ALM_PROXY),           false);
+
+        assertEq(vault.minVsr(),     1e27);
+        assertEq(vault.maxVsr(),     1e27);
+        assertEq(vault.depositCap(), 0);
+
+        assertEq(ctx.rateLimits.getCurrentRateLimit(takeKey),     0);
+        assertEq(ctx.rateLimits.getCurrentRateLimit(transferKey), 0);
+
+        executeAllPayloadsAndBridges();
+
+        assertEq(vault.hasRole(vault.DEFAULT_ADMIN_ROLE(), Ethereum.SPARK_PROXY), true);
+        assertEq(vault.hasRole(vault.SETTER_ROLE(), Ethereum.ALM_OPS_MULTISIG),   true);
+        assertEq(vault.hasRole(vault.TAKER_ROLE(), Ethereum.ALM_PROXY),           true);
+
+        assertEq(vault.minVsr(),     minVsr);
+        assertEq(vault.maxVsr(),     maxVsr);
+        assertEq(vault.depositCap(), depositCap);
+
+        assertEq(ctx.rateLimits.getCurrentRateLimit(takeKey),     type(uint256).max);
+        assertEq(ctx.rateLimits.getCurrentRateLimit(transferKey), type(uint256).max);
+
+        _testVaultTakeIntegration({
+            asset:      vault.asset(),
+            vault:      vault_,
+            rateLimit:  type(uint256).max,
+            takeAmount: amount
+        });
+
+        _testTransferAssetIntegration({
+            token:             vault.asset(),
+            destination:       vault_,
+            controller_:       Ethereum.ALM_CONTROLLER,
+            expectedRateLimit: type(uint256).max,
+            transferAmount:    amount
+        });
```

Comment:
Lets also prank as the ALM_OPS_MULTISIG here and do a setVsr, warp, then show nowChi changing (can just use assertGt)